src/components/views/elements/Field.js

```diff
@@ -144,7 +144,7 @@ export default class Field extends React.PureComponent {
     render() {
         const {
             element, prefix, onValidate, children, tooltipContent, flagInvalid,
-            ...inputProps} = this.props;
+            tooltipClassName, ...inputProps} = this.props;
```

Comment:
Maybe move `} = this.props;` to the next line?

Reply:
can't because then `...inputProps` needs a trailing comma because of our linter. When adding the comma though, the compiler says it is invalid ES6 because the rest parameter needs to be the last element.

Reply:
😭